mksyscall_windows.go

```diff
@@ -567,8 +571,9 @@ func (f *Fn) HelperCallParamList() string {
 
 // IsUTF16 is true, if f is W (utf16) function. It is false
 // for all A (ascii) functions.
-func (_ *Fn) IsUTF16() bool {
-	return true
+func (f *Fn) IsUTF16() bool {
+	s := f.DLLFuncName()
```

Comment:
Oops, was juggling too many copies of this file (there's gotta be a better vendoring solution than is currently being used). Fixed.